app/assets/v2/js/pages/onboard.js

```diff
@@ -0,0 +1,164 @@
+var current = 0;
+var words = [];
+
+$('.js-select2').each(function() {
+  $(this).select2();
+});
+
+var showTab = function(num) {
+  $($('.step')[num]).addClass('block').outerWidth();
+  $($('.step')[num]).addClass('show');
+
+  if (num == 0)
+    $('#prev-btn').hide();
+  else
+    $('#prev-btn').show();
+
+  if (num == ($('.step').length) - 1) {
+    $('#next-btn').html('Finish');
```

Comment:
is this going to be translated somehow?

Reply:
Ah don't think it will! @mbeacom ?

Reply:
As it sits currently, it won't. Should we wrap this in `gettext`?